kfs-cam/src/main/java/edu/arizona/kfs/module/cab/businessobject/PurchasingAccountsPayableDocument.java

```diff
@@ -0,0 +1,44 @@
+package edu.arizona.kfs.module.cab.businessobject;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import org.apache.commons.lang.StringUtils;
+import edu.arizona.kfs.module.cab.CabConstants;
+import org.kuali.kfs.module.cab.CabPropertyConstants;
+import org.kuali.rice.krad.service.BusinessObjectService;
+import org.kuali.kfs.module.purap.document.PaymentRequestDocument;
+import org.kuali.kfs.module.purap.document.VendorCreditMemoDocument;
```

Comment:
Please change the import to edu.arizona.kfs.module.purap.document.VendorCreditMemoDocument and edu.arizona.kfs.module.purap.document.PaymentRequestDocument to gain access to the Payment Method Code that was added.